src/components/CreateAccount/styles.scss

```diff
@@ -47,15 +47,15 @@
   align-items: center;
   flex-direction: column;
   justify-content: space-evenly;
+  height: 100%;
```

Comment:
Good point, I changed it because when this page is on bigger screen, there is some white all around the form. But I think the white is still better than possibly cutting out bottom of the page. :) I'll change it back.